### PR TITLE
Add Squirrel as generated tool

### PR DESCRIPTION
**Pull request by markusamshove**

### CODE REVIEW

**Comment by markusamshove** [comment]
I've updated the definitions to add all command line options of Squirrel.

I'm not exactly sure on how to design switches that take multiple arguments, like

```
shortcut-locations=
Comma-separated string of shortcut locations, e.g. 'Desktop,StartMenu'
```

Maybe you can help me @matkoch  :-)

**Comment by arodus** [comment]
@MarkusAmshove you can also use `List<?>` as a type. Take a look at [NuGet.json](https://github.com/nuke-build/nuke/blob/develop/build/specifications/NuGet.json#L265-L271) for an example.

**Comment by markusamshove** [comment]
Thanks @arodus 
In my optinion this should be done now.

The `process-start-args` isn't actually a list, they pass it to a `Process` as a single string argument.

File: build/specifications/Squirrel.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+          {
+            "name": "Uninstall",
+            "type": "bool",
+            "format": "--uninstall",
+            "help": "Uninstall the app the same dir as Update.exe."
+          },
+          {
+            "name": "Download",
+            "type": "string",
+            "format": "--download",
```
Comment (arodus) [inline build/specifications/Squirrel.json]: Missing `{value}` in format string.

File: build/specifications/Squirrel.json [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            "help": "Download the releases specified by the URL and write new results to stdout as JSON."
+          },
+          {
+            "name": "CheckForUpdate",
+            "type": "string",
+            "format": "--checkForUpdate",
```
Comment (arodus) [inline build/specifications/Squirrel.json]: Missing `{value}` in format string.

File: build/Build.GitFlow.cs [hunk 3]
```diff
@@ -23,38 +23,34 @@ partial class Build
             }
             else
             {
-                ReleaseFrom($"release/{GitVersion.MajorMinorPatch}");
```
Comment (arodus) [inline build/Build.GitFlow.cs]: Rebase onto develop please.

**Review by arodus — APPROVED** [review]

**Review by arodus — CHANGES_REQUESTED** [review]
Please rebase onto develop and fix the issues.

File: build/specifications/Squirrel.json (hunk 1 above)
Comment (markusamshove) [reply build/specifications/Squirrel.json]: Those don't take parameters, so I guess "properties" is the wrong category here.
Is there an action in another tool that I can take as example which adds a parameter to the tool, but doesn't take an argument itself?

File: build/Build.GitFlow.cs (hunk 3 above)
Comment (markusamshove) [reply build/Build.GitFlow.cs]: I've resolved the conflicts

**Review by arodus — APPROVED** [review]

**Pull request merged**